netlify.toml

```
@@ -1,5 +1,5 @@
 [context.production]
-  environment = { REACT_APP_SANITY_TAG="default", REACT_APP_SANITY_DATASET = "production", REACT_APP_API_URI = "https://regen-registry-server.herokuapp.com", GATSBY_SANITY_DATASET = "production", GATSBY_API_URI = "https://regen-registry-server.herokuapp.com", GATSBY_RECAPTCHA_SITE_KEY = "6LeqBFIaAAAAAPfLave4lUtLBKF2WYXinbaLNFqB", GATSBY_INTERCOM_APP_ID = "kn5di10s" }
+  environment = { REACT_APP_SANITY_TAG="default", REACT_APP_SANITY_DATASET = "production", REACT_APP_API_URI = "https://regen-registry-server.herokuapp.com", GATSBY_SANITY_DATASET = "production", GATSBY_API_URI = "https://regen-registry-server.herokuapp.com", GATSBY_RECAPTCHA_SITE_KEY = "6LeqBFIaAAAAAPfLave4lUtLBKF2WYXinbaLNFqB", GATSBY_INTERCOM_APP_ID = "kn5di10s", REACT_APP_BLOCK_EXPLORER="https://www.mintscan.io/regen" }
```

To work with the way `block-explorer.ts` is written, I think it needs to be this

```suggestion
  environment = { REACT_APP_SANITY_TAG="default", REACT_APP_SANITY_DATASET = "production", REACT_APP_API_URI = "https://regen-registry-server.herokuapp.com", GATSBY_SANITY_DATASET = "production", GATSBY_API_URI = "https://regen-registry-server.herokuapp.com", GATSBY_RECAPTCHA_SITE_KEY = "6LeqBFIaAAAAAPfLave4lUtLBKF2WYXinbaLNFqB", GATSBY_INTERCOM_APP_ID = "kn5di10s", REACT_APP_BLOCK_EXPLORER="https://www.mintscan.io" }
```